server/app/controllers/ErrorHandler.java

```diff
@@ -68,6 +69,11 @@ public CompletionStage<Result> onServerError(RequestHeader request, Throwable ex
       return CompletableFuture.completedFuture(Results.unauthorized());
     }
 
+    // Unwrap CompletionException
```

Comment:
This 'what' is pretty readable, can you document the 'why' also?

I'm trying to figure out if this changes the information revealed to the user in prod mode and also wondering what information that is.  An end user seeing an exception isn't the most useful thing (as compared to a nice broken robot) and we should be careful about leaking sensitive configuration information through exceptions.

Do you know how that changes (if at all) with this?   EG I know for a lot of error users see the general red screen with a hashcode that an IT manager could look for in the logs.

Reply:
This doesn't affect that visibility at all.  If something was shown before to a user, it still will be (and that seems to usually be a rather vague "oops an error occurred" 
![image](https://user-images.githubusercontent.com/5566205/178558616-8697e49c-bc8f-45d2-9b55-4a73eafa7790.png)
 
 
 All this does is unwrap any exceptions thrown inside a completable future, (which are rather unhelpfully wrapped in a completionexception), and shows the error message for underlying error that was thrown